Content.Shared/Chat/Prototypes/EmoteSoundsPrototype.cs

```diff
@@ -2,36 +2,49 @@
 using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Dictionary;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Array;
 
 namespace Content.Shared.Chat.Prototypes;
 
 /// <summary>
 ///     Sounds collection for each <see cref="EmotePrototype"/>.
 ///     Different entities may use different sounds collections.
 /// </summary>
-[Prototype]
-public sealed partial class EmoteSoundsPrototype : IPrototype
+[Prototype, Serializable, NetSerializable]
+public sealed partial class EmoteSoundsPrototype : IPrototype, IInheritingPrototype
 {
     [IdDataField]
     public string ID { get; private set; } = default!;
 
+    /// <inheritdoc/>
+    [ParentDataField(typeof(AbstractPrototypeIdArraySerializer<EmoteSoundsPrototype>))]
+    public string[]? Parents { get; private set; }
+
+    /// <inheritdoc/>
+    [AbstractDataField]
+    [NeverPushInheritance]
+    public bool Abstract { get; }
+
     /// <summary>
     ///     Optional fallback sound that will play if collection
     ///     doesn't have specific sound for this emote id.
     /// </summary>
     [DataField("sound")]
+    [AlwaysPushInheritance]
     public SoundSpecifier? FallbackSound;
 
     /// <summary>
     ///     Optional audio params that will be applied to ALL sounds.
     ///     This will overwrite any params that may be set in sound specifiers.
     /// </summary>
     [DataField("params")]
+    [AlwaysPushInheritance]
     public AudioParams? GeneralParams;
 
     /// <summary>
     ///     Collection of emote prototypes and their sounds.
     /// </summary>
-    [DataField("sounds", customTypeSerializer: typeof(PrototypeIdDictionarySerializer<SoundSpecifier, EmotePrototype>))]
+    [DataField(customTypeSerializer: typeof(PrototypeIdDictionarySerializer<SoundSpecifier, EmotePrototype>))]
+    [AlwaysPushInheritance]
     public Dictionary<string, SoundSpecifier> Sounds = new();
```

Comment:
while you're here you should replace the `string` and the custom type serializer with a `ProtoId<EmotePrototype>`. That'll also let you remove some of the imports off the top.